ui/src/pages/publications/[id]/index.tsx

```diff
@@ -395,7 +395,16 @@ const Publication: Types.NextPage<Props> = (props): React.ReactElement => {
                     <Components.PublicationContentSection id="references" title="References" hasBreak>
                         {references.map((reference) => (
                             <div key={reference.id} className="py-2">
-                                <Components.ParseHTML content={reference.text} />
+                                <Components.ParseHTML
+                                    content={
+                                        reference.location
+                                            ? reference.text.replaceAll(
```

Comment:
this  fails if the URL had HTML entities in it e.g.

https://doi.org/10.1002/(SICI)1096-9136(199801)15:1<53::AID-DIA521>3.0.CO;2-V

the reference.text has

dsfsd fsd sd d fdsd https://doi.org/10.1002/(SICI)1096-9136(199801)15:1&lt;53::AID-DIA521&gt;3.0.CO;2-V

so replaceAll doesnt match it

Reply:
Saved url will never have html entities because it's saved as plain text.

Reply:
replaceAll will replace exactly the extracted url. If the extracted url is broken, replaceAll will replace all the occurances as they are in the reference text

Reply:
reference.text has </> replaced by &lt;/&gt; so it will never match:

<p>https://doi.org/10.1002/(SICI)1096-9136(199801)15:1&lt;53::AID-DIA521&gt;3.0.CO;2-V</p>

Reply:
![image](https://user-images.githubusercontent.com/48569671/189334873-aa30bc6e-2bff-485c-8bd3-13d768c554cc.png)

Reply:
that is not what I'm saying, try 

https://doi.org/10.1234/<abc>